### PR TITLE
Create update.json for flexslider

### PR DESCRIPTION
**Pull request by bdukes**
flexslider has just released version 2.5

### CODE REVIEW

File: files/flexslider/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "flexslider",
+  "repo": "woothemes/FlexSlider",
+  "files": {
+    "include": ["jquery.flexslider.js", " jquery.flexslider-min.js", "flexslider.css", "fonts/*", "images/*"]
```
Comment (jimaek) [inline files/flexslider/update.json]: fonts and images need `./` in front of them to retain the directory structure

Comment (bdukes) [inline files/flexslider/update.json]: Fixed in b6459595bf39e1115e6445a9dcd3369b93bd4a56, thanks!

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@bdukes Thanks

**Pull request closed without merging**